crypto-ffi/src/core_crypto/logger.rs

```diff
@@ -275,7 +265,7 @@ fn set_logger_only_inner(logger: Logger) {
 /// Initializes the logger
 #[cfg(not(target_family = "wasm"))]
 #[uniffi::export]
-pub fn set_logger_only(logger: Logger) {
+pub fn set_logger(logger: Logger) {
     set_logger_only_inner(logger);
```

Comment:
Is `set_logger_only_inner` used anywhere else, or could we reasonably inline it here, now that the other `set_logger` call is gone?